docs/Installation.md

```diff
@@ -124,15 +124,15 @@ sudo install -v -m 755 ./tmpfiles.conf /usr/lib/tmpfiles.d/zonemaster.conf
 > previous version of Zonemaster-Backend).

```

Comment:
On line 123 it says "/etc/init.d/zm-centos.sh". Should that be kept?

Reply:
I guess we could assume that no one is in this case anymore and remove it.

Reply:
I removed the note about the old scripts